README.md

```diff
@@ -23,3 +23,7 @@ cd build
 cmake ..
 make
 ```
+## Disclaimer
+
+After changing the repository name or transfering ownership, run the "Update README and LaTeX Build" workflow from the Actions tab.
+Make sure the `.tex` file has the same name as the repository in order for it to build correctly!
```

Comment:
Now that we have this in python, we could also add a check for this that reports and error of this is not the case

Reply:
Yes